From 17aabbf049429d23eb5d493a0b1f455d84d80a27 Mon Sep 17 00:00:00 2001
From: Shuguang Gong <shuguang.gong@intel.com>
Date: Tue, 31 Mar 2015 15:58:13 +0800
Subject: [PATCH 350/351] ov2685: improve EV contrast

Change-Id: I07dc5e6c37e9ca3d4cb0586af362f159c4f94c2b
Tracked-On: https://hsdes.intel.com/home/
default.html#article?id=1503940920
Signed-off-by: Shuguang Gong <shuguang.gong@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/48991
Approver: Jing Tao <jing.tao@intel.com>
Build: IMIN-legacy (SC)
Integrator: Wei Tao <wei.tao@intel.com>
Build: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
Maintainer: Wei Tao <wei.tao@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/ov2685.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov2685.c b/drivers/external_drivers/camera/drivers/media/i2c/ov2685.c
index 26241d7..7821b34 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov2685.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov2685.c
@@ -574,12 +574,12 @@ static int ov2685_s_ev(struct v4l2_subdev *sd, int value)
 	switch (value) {
 	case -2:
 		ret = ov2685_write_reg(client, OV2685_8BIT,
-			OV2685_REG_WPT, 0x32);
+			OV2685_REG_WPT, 0x2a);
 		if (ret)
 			return -EINVAL;
 
 		ret = ov2685_write_reg(client, OV2685_8BIT,
-			OV2685_REG_BPT, 0x28);
+			OV2685_REG_BPT, 0x20);
 		break;
 	case -1:
 		ret = ov2685_write_reg(client, OV2685_8BIT,
@@ -619,12 +619,12 @@ static int ov2685_s_ev(struct v4l2_subdev *sd, int value)
 		break;
 	case 2:
 		ret = ov2685_write_reg(client, OV2685_8BIT,
-			OV2685_REG_WPT, 0x62);
+			OV2685_REG_WPT, 0x6a);
 		if (ret)
 			return -EINVAL;
 
 		ret = ov2685_write_reg(client, OV2685_8BIT,
-			OV2685_REG_BPT, 0x58);
+			OV2685_REG_BPT, 0x60);
 		if (ret)
 			return -EINVAL;
 		break;
-- 
1.9.1

